### ISSUE TITLE
[BUG] Can't delete entries from bad files anymore

### ISSUE THREAD

**Issue by quorn23** [aetaric/checkrr#78]
**Describe the bug**
Can't delete entries from bad files anymore. 

**To Reproduce**
1. Mark a bad file
2. Click Delete selected rows
3. Confirm deletion
4. Webui refreshes but bad file entry is still

**Expected behavior**
Removing the Bad entry.

**Desktop (please complete the following information):**
 - OS: Latest docker on Debian
 - checkrr Version: [v3.2.1]

**Additional context**
Neither checkrr.log or the docker.log throw any error i could provide.

**Docker Compose**
```
  checkrr:
    container_name: checkrr
    image: aetaric/checkrr:latest
    volumes:
      - ${DOCKERCONFDIR}/checkrr/checkrr.yaml:/etc/checkrr.yaml
      - ${DOCKERCONFDIR}/checkrr/checkrr.db:/checkrr.db
      - ${DOCKERCONFDIR}/checkrr/checkrr.log:/checkrr.log
      - ${DOCKERDATADIR}:/data
    ports:
      - 8585:8585
    restart: on-failure
    logging: *default-logging
```

**Comment by quorn23** [comment]
[https://img.amoklauf.ch/gOjE3/sUheyuBa96.gif/raw](https://img.amoklauf.ch/gOjE3/sUheyuBa96.gif/raw)
Here a gif of the bug visually.

**Comment by aetaric** [comment]
As an FYI, this issue was triaged on discord and I already have a rough idea why it's happening. I plan to look into a fix this weekend. It will be patched in 3.2.2 once that's out.
